src/main.ts

```diff
@@ -1,5 +1,6 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
 import {
+  faArrowsRotate,
```

Comment:
use the same line

Reply:
yes, I don't know why the auto format put it there ^^